modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -304,8 +311,13 @@ If you produce to a topic that doesn't exist, the topic will be created with def
 ---
 
 
+// tag::cluster_id[]
```

Comment:
We forgot to update the google sheet but cluster_id won't be exposed

Reply:
thanks, I've removed it from Cloud docs